architecture/routing.adoc

```diff
@@ -10,17 +10,38 @@
 toc::[]
 
 == Overview
-Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information directly to the router (HTTP, HTTPS, and TLS with SNI). With a service, the router uses the service selector to watch the endpoints that support the service, bypassing its logic and replacing it with the router's own. Routers subscribe to a configuration and automatically update themselves with any changes, and may be containerized or virtual (converting those changes to API calls to a system like an F5).
+Routers provide external DNS mapping and load balancing to services over the
+following protocols that pass distinguishing information directly to the router:
```

Comment:
Changed to: "Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information directly to the router. Routers support the following protocols:"